test/unit/auth/auth.spec.ts

```diff
@@ -312,39 +312,17 @@ AUTH_CONFIGS.forEach((testConfig) => {
         });
       });
 
-      describe('Auth.forTenant()', () => {
-        it('should cache the returned TenantAwareAuth', () => {
-          const tenantAwareAuth1 = (auth as Auth).forTenant('tenantId1');
-          const tenantAwareAuth2 = (auth as Auth).forTenant('tenantId2');
-          expect((auth as Auth).forTenant('tenantId1')).to.equal(tenantAwareAuth1);
-          expect((auth as Auth).forTenant('tenantId2')).to.equal(tenantAwareAuth2);
-          expect(tenantAwareAuth1).to.not.be.equal(tenantAwareAuth2);
-          expect(tenantAwareAuth1.tenantId).to.equal('tenantId1');
-          expect(tenantAwareAuth2.tenantId).to.equal('tenantId2');
-        });
-      });
-    } else {
-      // Run tests for TenantAwareAuth.
-      describe('forTenant()', () => {
-        const invalidTenantIds = [null, NaN, 0, 1, true, false, '', [], [1, 'a'], {}, { a: 1 }, _.noop];
-        invalidTenantIds.forEach((invalidTenantId) => {
-          it('should throw given invalid tenant ID: ' + JSON.stringify(invalidTenantId), () => {
-            expect(() => {
-              const testAuth = new Auth(mockApp);
-              return testAuth.forTenant(invalidTenantId as any);
-            }).to.throw('The tenant ID must be a valid non-empty string.');
-          });
-        });
-
-        it('should return a TenantAwareAuth with the expected tenant ID', () => {
-          const testAuth = new Auth(mockApp);
-          expect(testAuth.forTenant(TENANT_ID).tenantId).to.equal(TENANT_ID);
+      describe('tenantManager()', () => {
+        it('should return the expected tenant manager instance', () => {
```

Comment:
The description is not very informative. Maybe 'should return TenantManager with same attributes'?

Reply:
I changed to:
`should return a TenantManager with the expected attributes`